src/Salt/SiteBundle/Controller/CommentsController.php

```diff
@@ -166,4 +167,60 @@ private function apiResponse($data, $statusCode = 200): JsonResponse
 
         return JsonResponse::fromJsonString($json, $statusCode);
     }
+    
+    /**
+     * @Route("/salt/case/export_comment/{itemType}/{id}/comment.csv", name="export_comment_file")
+     *
+     * @param int $id
+     * 
+     * @param string $itemType
+     * 
+     * @param Request $request
+     *
+     * @return Response
+     */
+    public function exportCommentAction(string $itemType, int $id,Request $request)
+    {        
+        $url=$request->getBaseUrl('_route').'/cftree/'.$itemType.'/'.$id;
```

Comment:
Is the URL column mandatory? can't we just export the item type and the item id?.
We would need to refactor if the URL changes the format.

Reply:
Yes, In story acceptance criteria mentioned that column. As write now we are not storing url in db, we have to create manually.